src/scss/_chromecastButton.scss

```diff
@@ -6,6 +6,7 @@ $icon-chromecast-casting--hover: 'images/ic_cast_connected_white_24dp.png' !defa
 
 // Sizes
 $chromecast-icon-size: 12px !default;
+$chromecast-button-lg-width: 5em !default;
```

Comment:
I think we will need to find a way to style this by keeping the width flexible so that it can grow with the width of the "Cast" label. This value works fine in English, but if the label is translated into another language, the string could be a lot longer and run out of bounds:

<img width="151" alt="image" src="https://user-images.githubusercontent.com/1326722/187546567-cfe488d7-da17-49d9-bf81-e3aca31706ba.png">

We may be able to do something by setting `width: auto` and adding a little padding around the edges of the button, as well as some margin in between the icon and label.